content/en/releases/checklist-for-victims-of-a-data-breach.md

```diff
@@ -0,0 +1,187 @@
+
+### File a criminal complaint with the police for data leak and/or extortion
+
+*   You can report an offence online on the [website of the Finnish Police](https://www.poliisi.fi/crimes/reporting_an_offence_online). The service is available every day from 6 to 22:45. If you are not able to do a report immediately, try again later. If you have received a ransom note, include a screenshot of it as an attachment.
+*   If you can't report an offence online, do it at your local police station.
+*   Police has instructed to not call the Emergency Response Centers.
+
+### Report the incident to National Cyber Security Centre 
+
+*   You can file a report with the [form provided by Finnish Cyber Security Centre](https://www.kyberturvallisuuskeskus.fi/en/report)
+*   Based on the notifications, the NCSC-FI investigates security violations and threats to information security on network services, communications services and added value services, collects information on these incidents, and helps organisations with instructions and sometimes with technical assistance.
+
+### Request personal prohibition of automated credit
+
+*   This ensures that nobody can take a loan in your name. Finland has two companies maintaining credit registries: Asiakastieto and Bisnode. We recommend you request for a credit ban from both. Note that the credit ban lasts for two years at a time and you will have to renew it if you wish to continue it.
+*   Asiakastieto will charge 19,95€ for the ban. [You can request it online from Asiakastieto](https://www.asiakastieto.fi/omatieto/en/tuotteet/oma-luottokielto) (currently only in Finnish)
+*   Bisnode credit ban costs 15,95€. [You can request it online from Bisnode](https://my.bisnode.fi/protection) (currently only in Finnish)
+*   A credit ban can prevent frauds using your personal information. Using the personal identification number and other personal information, a criminal may be able to make small purchases or open other lines of credit in your name. This voluntary credit ban does not prevent you from buying in installments or requesting other credit. You'll receive a separate certificate, which you can personally use to prove your creditworthiness despite the voluntary credit ban.
+
+### Request prohibition for address change with Posti free-of-charge
+
+*   You can prevent address changes by activating [address change protection from Posti](https://www.posti.fi/fi/asiakastuki/tiedotteet/postin-ohje-psykoterapiakeskus-vastaamon-tietomurtoon-liittyen-nain-teet). (currently only in Finnish)
+*   The ban is requested from [Posti's online customer service portal](https://www.posti.fi/omatpalvelut/feedback?lang=fi) under "Address change protection, Feedback about Mail forwarding or Nearby Mailbox."
+*   The protection prevents address changes and mail delivery suspensions as long as the protection is enabled. A criminal can't then order deliveries or invoices under your name to another address.
```

Comment:
The protection prevents address changes and mail delivery suspensions as long as the protection is enabled. A criminal can not then order deliveries or invoices under your name to another address.